firstvoices/backend/views/sites_views.py

```diff
@@ -96,3 +96,76 @@ def list(self, request, *args, **kwargs):
         ]
 
         return Response(data)
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="A list of language sites a given user is a member of, "
+        "grouped by language. If there are no accessible sites or "
+        "the user does not have membership to any site then the "
+        "list will be empty. Sites with no specified language "
+        "will be grouped under 'Other'.",
```

Comment:
My only question about this is whether we want the sites grouped by language or just in a plain list. @gmcauliffe do you have thoughts?

Reply:
I chatted with @gmcauliffe and it would be better if this were all in one list. No need to group by language

Reply:
I have removed the grouping by language for this API. Thanks for the feedback!